src/progress.rs

```diff
@@ -432,6 +432,7 @@ impl ProgressState {
     /// Finishes the progress bar and leaves the current message and progress.
     pub fn abandon(&mut self) {
         self.update_and_draw(|state| {
+            state.draw_next = state.pos;
```

Comment:
This is not very obvious. Should we just always draw in state DoneVisible? Or give the closure a return type that will force it to draw?